paws_collector.js

```diff
@@ -47,17 +84,31 @@ class PawsCollector extends AlAwsCollector {
     static load() {
         return new Promise(function(resolve, reject){
             AlAwsCollector.load().then(function(aimsCreds) {
-                getDecryptedPawsCredentials(function(err, pawsCreds) {
-                    if (err){
-                        reject(err);
-                    } else {
-                        resolve({aimsCreds : aimsCreds, pawsCreds: pawsCreds});
-                    }
-                });
-            })
-        })
+                if(process.env.paws_auth_type === 's3object'){
```

Comment:
I think we can refactor this if statement to and call `getDecryptedPawsCredentials` only once.

Reply:
Let's use `switch` statement instead of `if` here? To simplify adding new auth type in the future, like.
```
switch (process.env.paws_auth_type) {
  case 's3object':
    credsBuffer = getPawsCredsFile();
    break;
  default:
    credsBuffer = Buffer.from(process.env.paws_api_secret, 'base64');
}

getDecryptedPawsCredentials(credsBuffer);
```